eox_core/api/v1/tests/integration/test_views.py

```diff
@@ -0,0 +1,434 @@
+        - The status code is 404.
+        - The response contains an error message.
+        """
+        response = self.get_user_in_tenant(self.tenant_x, {param: value})
+
+        response_data = response.json()
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+        self.assertIn("detail", response_data)
+        self.assertEqual(
+            response_data["detail"],
+            f"No user found by {{'{param}': '{value}'}} on site {self.tenant_x['domain']}.",
+        )
+
+    def test_update_user_in_tenant_success(self) -> None:
+        """
+        Test updating a user in a tenant.
+
+        Open edX definitions tested:
+        - `get_edxapp_user`
+        - `get_user_profile`
+        - `check_edxapp_account_conflicts`
+        - `get_user_read_only_serializer`
+
+        Expected result:
+        - The status code is 200.
+        - The user is updated successfully in the tenant with the provided data.
+        """
+        data = get_fake_user_data()
+        self.create_user_in_tenant(self.tenant_x, data)
+        updated_data = {
+            "username": data["username"],
+            "email": data["email"],
+            "fullname": fake.name(),
+            "password": fake.password(),
+            "is_active": False,
+            "mailing_address": fake.address(),
+            "year_of_birth": int(fake.year()),
+            "gender": fake.random_element(elements=("m", "f", "o")),
+            "level_of_education": fake.random_element(
+                elements=("p", "m", "b", "a", "hs", "jhs", "el", "none", "other")
+            ),
+            "city": fake.city(),
+            "goals": fake.text(),
+            # These fields are not updated
+            # "phone_number": fake.msisdn(),
+            # "country": fake.country_code(),
+            # "bio": fake.text(),
+        }
+
+        response = self.update_user_in_tenant(self.tenant_x, user_data=updated_data)
+
+        response_data = response.json()
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response_data["username"], data["username"])
+        self.assertEqual(response_data["email"], data["email"])
+        self.assertEqual(response_data["name"], updated_data["fullname"])
+        self.assertEqual(response_data["mailing_address"], updated_data["mailing_address"])
+        self.assertEqual(response_data["year_of_birth"], updated_data["year_of_birth"])
+        self.assertEqual(response_data["gender"], updated_data["gender"])
+        self.assertEqual(response_data["level_of_education"], updated_data["level_of_education"])
+        self.assertEqual(response_data["goals"], updated_data["goals"])
+        self.assertFalse(response_data["is_active"])
+        # self.assertEqual(response_data["phone_number"], updated_data["phone_number"])
+        # self.assertEqual(response_data["country"], updated_data["country"])
+        # self.assertEqual(response_data["bio"], updated_data["bio"])
```

Comment:
Do we need these lines?

Reply:
Well, I wouldn't think so. But since you remind me, it would be good to check why those fields are not being updated. I'm really not sure if that should be the expected behavior.

Reply:
I don't know either. Maybe there's more information right in the code. You could do a quick check in case it's something we should fix soon.

Reply:
According to this setting, these fields should be updated. 
https://github.com/eduNEXT/eox-core/blob/a8ae7152a22084b7188e18550086da9d0e52cbb7/eox_core/settings/common.py#L48

Reply:
Can you replicate this elsewhere? Can we create an internal ticket to address this issue?

Reply:
I just tested on the Redwood stage and the same thing happens. I will open an issue